main.tf

```diff
@@ -224,6 +224,28 @@ resource "aws_ec2_transit_gateway_vpc_attachment" "default" {
   transit_gateway_id                              = var.transit_gateway_id
   vpc_id                                          = aws_vpc.default.id
   tags                                            = { Name = var.name }
+
+  lifecycle {
+    ignore_changes = [
+      transit_gateway_default_route_table_association, transit_gateway_default_route_table_propagation
+    ]
+  }
+}
+
+resource "aws_ec2_transit_gateway_vpc_attachment_accepter" "default" {
+  provider = aws.transit_gateway_account
+  count    = anytrue(var.networks[*].tgw_attachment) ? 1 : 0
+
+  transit_gateway_attachment_id                   = aws_ec2_transit_gateway_vpc_attachment.default[count.index].id
+  transit_gateway_default_route_table_propagation = true
+  tags                                            = var.tags
+  depends_on                                      = [aws_ec2_transit_gateway_vpc_attachment.default[0]]
+
+  lifecycle {
+    ignore_changes = [
+      transit_gateway_default_route_table_association, transit_gateway_default_route_table_propagation
```

Comment:
is this ignore changes needed it's not in the transit gatewy module code?

Reply:
Removed!